File: 프로그래머스/unrated/181946. 문자열 붙여서 출력하기/문자열 붙여서 출력하기.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    input = line.split(' ');
}).on('close', function () {
    str1 = input[0];
    str2 = input[1];
    
    if(str1.length >= 1 && str2.length <= 10){
        console.log(`${str1}${str2}`);
    }else {
        console.log("문자열의 제한은 1 <= str1, str2 <= 10 입니다.")
    }
});

File: 프로그래머스/unrated/181946. 문자열 붙여서 출력하기/문자열 붙여서 출력하기.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.next();
        String b = sc.next();
        
        try{
            int aLeng = a.length();
            int bLeng = b.length();
            
            if(1 <= aLeng && bLeng <= 10){
                String str1 = a;
                String str2 = b;
                
                System.out.println(str1+str2);
            }
        }finally {
            sc.close();
        }
    }
}